src/api/tests/test_ingest.py

```
@@ -427,3 +428,72 @@ def test_benthicpqt_ingest(
+    invert_belt_transect_width_1m,
+    invert_size_bin_1,
+    current1,
+    tide1,
+    reef_slope1,
+    relative_depth1,
+    visibility1,
+    site1,
+    site2,
+    management1,
+):
+    new_records, output = utils.ingest(
```

Ruff (0.15.13)

[warning] Unpacked variable `output` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)